photon-client/src/components/cameras/CameraCalibrationInfoCard.vue

```diff
@@ -289,6 +311,33 @@ const calibrationImageURL = (index: number) =>
       </v-data-table>
     </v-card-text>
   </v-card>
+
+  <v-dialog v-model="confirmRemoveDialog.show" width="600">
+    <v-card color="surface" dark>
+      <v-card-title>Delete Calibration</v-card-title>
+      <v-card-text class="pt-0">
+        Are you sure you want to delete the calibration for {{ confirmRemoveDialog.vf.resolution.width }}x{{
+          confirmRemoveDialog.vf.resolution.height
+        }}?
+        <v-card-actions class="pt-5 pb-0 pr-0" style="justify-content: flex-end">
+          <v-btn
+            :variant="theme.global.name.value === 'LightTheme' ? 'elevated' : 'outlined'"
+            color="buttonPassive"
```

Comment:
```suggestion
            color="primary"
```
For consistency with other delete dialogs.